BGAnimations/ScreenEvaluation common/PerPlayer/ItlFile.lua

```diff
@@ -0,0 +1,162 @@
+local player = ...
+local pn = ToEnumShortString(player)
+
+local year = Year()
+local month = MonthOfYear()+1
+local day = DayOfMonth()
+
+local IsEventActive = function()
+	-- The file is only written to while the event is active.
+	-- These are just placeholder dates.
+	local startTimestamp = 20220331
+	local endTimestamp = 20220330
+
+	local today = year * 10000 + month * 100 + day
+
+	return startTimestamp <= today and today <= endTimestamp
+end
+
+local style = GAMESTATE:GetCurrentStyle()
+local game = GAMESTATE:GetCurrentGame()
+
+if (SL.Global.GameMode == "Casual" or
+		GAMESTATE:IsCourseMode() or
+		not IsEventActive() or
+		(game and game:GetName() ~= "dance") or
+		(style and style:GetName() ~= "single" and style:GetName() ~= "versus")) then
```

Comment:
When would game or style be nil?

Reply:
Ehhh it probably wouldn't be. I kind of copied this from elsewhere but I don't think that in itself was correct. Removed.